nrf5/hal/hal_temp.c

```diff
@@ -25,18 +25,36 @@
  */
 
 #include <stdio.h>
+#include <stdint.h>
 #include "mphalport.h"
 #include "hal_temp.h"
 
+#if BLUETOOTH_SD
```

Comment:
it all depends if it's OK for us to have SD_ functions inside the HAL layer. My vote is for a SD agnostic pure HAL layer